File: CS/CustomResource.cs
namespace WindowsApplication1 {
    public class CustomResource {
        private string m_name;
        private int m_res_id;
        private string m_PostCode;
        private string m_Address;

        public string Name { get { return m_name; } set { m_name = value; } }
        public int ResID { get { return m_res_id; } set { m_res_id = value; } }
        public string PostCode { get { return m_PostCode; } set { m_PostCode = value; } }
        public string Address { get { return m_Address; } set { m_Address = value; } }

        public CustomResource() {
        }
    }
}

File: VB/CustomResource.vb
Namespace WindowsApplication1
    Public Class CustomResource
        Private m_name As String
        Private m_res_id As Integer
        Private m_PostCode As String
        Private m_Address As String

        Public Property Name() As String
            Get
                Return m_name
            End Get
            Set(ByVal value As String)
                m_name = value
            End Set
        End Property
        Public Property ResID() As Integer
            Get
                Return m_res_id
            End Get
            Set(ByVal value As Integer)
                m_res_id = value
            End Set
        End Property
        Public Property PostCode() As String
            Get
                Return m_PostCode
            End Get
            Set(ByVal value As String)
                m_PostCode = value
            End Set
        End Property
        Public Property Address() As String
            Get
                Return m_Address
            End Get
            Set(ByVal value As String)
                m_Address = value
            End Set
        End Property

        Public Sub New()
        End Sub
    End Class
End Namespace